project-base/tests/ShopBundle/Functional/Model/Product/Search/ProductSearchExportRepositoryTest.php

```diff
@@ -1,80 +0,0 @@
-<?php
-
-namespace Tests\ShopBundle\Functional\Model\Product\Search;
-
-use Shopsys\FrameworkBundle\Component\Domain\Domain;
-use Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportRepository;
-use Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportWithFilterRepository;
-use Tests\ShopBundle\Test\TransactionFunctionalTestCase;
-
-class ProductSearchExportRepositoryTest extends TransactionFunctionalTestCase
```

Comment:
If this test is useful, please don't remove it and rewrite it for the `ProductSearchExportWithFilterRepository`